stdlib/2and3/colorsys.pyi

```diff
@@ -0,0 +1,10 @@
+# Stubs for colorsys
+
+from typing import Tuple
+
+def rgb_to_yiq(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
```

Comment:
True; then mypy is making a shortcut by having `complex` being `Union[complex, float]`.